packages/runtime/container-runtime-definitions/src/snapshot.ts

```diff
@@ -0,0 +1,69 @@
+/*!
```

Comment:
this should all remain in container runtime. this should only be for interfaces we expect external developers to take a dependency on

Reply:
Moved this file to container-runtime package.